docs/snapshotting/snapshot-support.md

```diff
@@ -128,6 +128,16 @@ The snapshot functionality is still in developer preview due to the following:
 
 ### Limitations
 
+- Currently on aarch64 platforms only lower 128 bits of any register are
+  saved due to the limitations of `get/set_one_reg` from `kvm-ioctls` crate
+  that Firecracker uses to interact with KVM. This creates an issue with
+  newer aarch64 CPUs with support for registers with width greater than
+  128 bits, because these registers will be truncated before being stored
+  in the snapshot. This can lead to uVM failure if restored from such snapshot.
+  Because registers wider then 128 bits are usually used in SVE instructions,
+  the best way to mitigate this issue is to ensure that the software run in
+  uVM does not use SVE instructions. Alternative way is to use [CPU templates](../cpu_templates/cpu-templates.md)
```

Comment:
We should add "before taking a snapshot", otherwise it sounds like we don't support SVE workloads at all.